InternxtDesktop/Views/Backup/FolderSelectorView.swift

```diff
@@ -82,9 +92,10 @@ struct FolderSelectorView: View {
                         do {
                             try doBackup()
                         } catch {
-                            print("Error \(error.reportToSentry())")
+                            // show error in UI
+                            showErrorDialog(message: error.localizedDescription)
```

Comment:
Remember to report errors to Sentry when catched by the last consumer, in this case, is here, displaying the error in the UI doesn't allow us to track it